source/plugin/basics/events.rst

```diff
@@ -171,6 +171,74 @@ Example: Listen for Custom Event
         event.getReceipient().sendMessage(ChatTypes.CHAT, "PM from " + senderName + ": " + event.getMessage());
     }
 
+Dynamic Registering and Unregistering of Event Handlers
+=======================================================
+
+Some Plugins may wish to dynamically register or unregister an event handler. In that case the event handler is not a method annotated with ``@Subscribe`` but a class implementing the ``EventHandler`` interface. This event handler can then be registered by calling the event managers ``register()`` method, which accepts a reference to the plugin, the ``Class`` of events handled and the handler himself.
+
+The exemplary creation and registration of a handler for an imaginary ``FLARDImplosionEvent`` is demonstrated below.
+
+Example: Event Handler class
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: java
+
+    public class FLARDHandler implements EventHandler<FLARDImplosionEvent> {
+
+        public void handle(FLARDImplosionEvent event) throws Exception {
+            if (event.canPreventCatastrophy()) {
+                event.setRelieved(true);
+            } else {
+                throw new ImpendingDoomException();
+            }
+        }
+    }
+
+Example: Dynamically Registering the Event Handler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: java
+
+    @Plugin(...)
+    public class FLARDPlugin {
+        ...
+        private FLARDHandler handler;
+
+        public void registerFLARDHandler() {
+            this.handler = new FLARDHandler();
+            game.getEventManager().register(this, FLARDImplosionEvent.class, this.handler);
+        }
+        ...
+    }
+
+.. note::
+
+        There is also a ``register()`` method accepting an ``Order`` in addition to an ``EventHandler``, which can be used to control when the handler will be called. (See also `About @Subscribe`_).
+        However, dynamically registered event handlers will be called no matter if the event is cancelled or not. If the handler is intented to not react to cancelled events, an according check must be performed in the ``handle()`` method.
```

Comment:
intented => intended. Phrasing is awful anyway. Try "If the handler is *not* intended to react to cancelled events, a corresponding check must be performed in the ``handle()`` method."